index.html

```diff
@@ -24,65 +24,276 @@
-            <a href="https://www.linkedin.com//"
-              ><img
-                class="l-social-media"
-                src="img/Vector (1).svg"
-                alt="linkedIn social media logo"
-            /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img
-                class="m-social-media"
-                src="img/Linkedin icon.svg"
-                alt="medium"
-            /></a>
-          </li>
-          <li>
-            <a href="https://github.com/tomdan-ai"
-              ><img
-                class="g-social-media"
-                src="img/Vector (3).svg"
-                alt="github logo"
-            /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img
-                class="v-social-media"
-                src="img/Vector (4).svg"
-                alt="vector logo"
-            /></a>
-          </li>
-        </ul>
-      </div>
+          <h1>
+            I'm Tom<br />
+            Glad to see you!
+          </h1>
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+        </div>
+        <div class="connect">
+          <a href="#">LET'S CONNECT</a>
+          <ul class="social-icons">
+            <li>
+              <a href="https://twitter.com/"
+                ><img
+                  class="t-social-media"
+                  src="img/Vector.svg"
+                  alt="twitter social media logo"
+              /></a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com//"
+                ><img
+                  class="l-social-media"
+                  src="img/Vector (1).svg"
+                  alt="linkedIn social media logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="m-social-media"
+                  src="img/Linkedin icon.svg"
+                  alt="medium"
+              /></a>
+            </li>
+            <li>
+              <a href="https://github.com/tomdan-ai"
+                ><img
+                  class="g-social-media"
+                  src="img/Vector (3).svg"
+                  alt="github logo"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  class="v-social-media"
+                  src="img/Vector (4).svg"
+                  alt="vector logo"
+              /></a>
+            </li>
+          </ul>
+        </div>
       </section>
     </main>
+    <!-- Tonic -->
+
+    <section class="grid">
+      <div class="work-card flex-column">
+        <img src="img/IMG1.png" alt="Tonic" />
+        <div class="work-card1">
+          <h2 class="tonic-class">Tonic</h2>
```

Comment:
- [Optional ] Please make sure that the last card in the work section has the card shape as in the design, and as other cards
![image](https://user-images.githubusercontent.com/74744787/219805577-408bf2a3-ea01-4331-b495-51997e6e1320.png)